types/mjml-react/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for mjml-react 1.0
+// Type definitions for mjml-react 2.0
```

Comment:
@adamth are the types backwards compatible, i.e no breaking changes for the types between 1.0 and 2.0?

Reply:
It appears that the only change for the major version bump was updating the babel runtime https://github.com/wix-incubator/mjml-react/commit/7894e20e52db6340e8bad2cbb731f6a3f2430db4
So no breaking changes 🎉